client/store/modules/repositories/index.js

```diff
@@ -8,6 +8,8 @@ const state = {
   items: {},
   search: '',
   showPinned: false,
+  tags: [],
+  selectedTags: [],
```

Comment:
Since this is within repositories module we should have the notion of what selectedTags refers to. Consider renaming to tagFilter (another potential solution would be to nest it within filter object but then we need to move other ones aswell)